core/src/main/scala/com/banno/kafka/RecordStream.scala

```diff
@@ -205,6 +212,21 @@ object RecordStream {
         ) {
           override protected def nextOffsets(x: A) = topical.nextOffset(x)
           override def records: Stream[F, A] = stream.records.flatMap(chunked)
+          override def processingAndCommitting[B](
+              maxRecordCount: Long = 1000L,
+              maxElapsedTime: FiniteDuration = 60.seconds,
```

Comment:
I don't think default args in an anonymous implementation will do anything but maybe get out of sync?

Reply:
Yeah these default args everywhere are a bit weird. Removed from the 2 impls in https://github.com/Banno/kafka4s/pull/846/commits/43345d5b72fc96298a68731313f2ff6269c8a92d.